moneyed/localization.py

```diff
@@ -197,6 +197,7 @@ def format(self, money, include_symbol=True, locale=DEFAULT,
 _sign(DEFAULT, moneyed.BIF, prefix='FBu')
 _sign(DEFAULT, moneyed.BMD, prefix='BD$')
 _sign(DEFAULT, moneyed.BND, prefix='B$')
+_sign(DEFAULT, moneyed.BOB, suffix=' Bs')
```

Comment:
There is no space for other currencies at the beginning. Is there something exceptional for BOB?